ATDM:  cts1empire not working on eclipse

## Question
@e10harvey, about a week ago (I think it was 2/3 or 4) our deploys of Trilinos on eclipse started failing with the following error:
```
00:02:57.481  Hostname 'eclipse-login1' matches known ATDM host 'eclipse-login1' and system 'cts1empire'
00:02:57.481  Setting compiler and build options for build-name 'Deploy-develop-eclipse-intel-opt-openmp-static'
00:02:57.481  Using cts1empire toss3 compiler stack INTEL-18.0.2_OPENMPI-4.0.1 to build RELEASE code with Kokkos node type OPENMP
00:03:01.711  ninja -C .  -j 16 ./all
00:03:01.711  ninja: Entering directory `.'
00:03:03.213  ninja: error: '/projects/sparc/tpls/cts1-bdw/netcdf-4.7.0/24baa07a3fa1ff9dbc8e70dc591ebbdec56783b2/cts1-bdw_intel-19.0.5_openmpi-4.0.1/lib/libnetcdf.a', needed by 'packages/seacas/libraries/ioss/src/main/io_shell', missing and no known rule to make it
00:03:03.213  make: *** [all] Error 1
00:03:03.213  make: Target `default_target' not remade because of errors.
```

I tried selecting the `intel-19` environment from `cts1` instead of `cts1empire`, but ran into more or less the same thing:
```
Hostname 'eclipse-login7' matches known ATDM host 'eclipse-login7' and system 'cts1'
Setting compiler and build options for build-name 'intel-19-opt-openmp-static'
Using cts1 toss3 compiler stack INTEL-19.0.4_OPENMPI-4.0.3 to build RELEASE code with Kokkos node type OPENMP
ninja -C .  -j 16 ./all
ninja: Entering directory `.'
ninja: error: '/projects/sparc/tpls/cts1-bdw/netcdf-4.7.0/1710af73d9e61434ec053613ec3764075718a9a5/cts1-bdw_intel-19.0.4_openmpi-4.0.3/lib/libnetcdf.a', needed by 'packages/seacas/libraries/ioss/src/main/io_modify', missing and no known rule to make it
make: *** [/home/jmgate/urgh/trilinos/src/Trilinos/cmake/tribits/core/utils/NinjaMakefileCommon.make:20: all] Error 1
make: Target 'default_target' not remade because of errors.
```

Are other builds on eclipse running into similar things?  Do we need to look into updating the `cts1` environment to use the latest `sparc-dev` module?  It sounds like EMPIRE has hacked around the Intel 19 compiler bug, so we may be able to ditch `cts1empire` and bring `cts1` up to date with whatever SPARC is using now.  Maybe.

--- a/cmake/std/atdm/cts1empire/environment_new.sh
+++ b/cmake/std/atdm/cts1empire/environment_new.sh
@@ -72,7 +72,7 @@ export SUPERLUDIST_ROOT=${sparc_tpl_base}/superlu_dist-5.4.0/a3121eaff44f7bf7d44
 
 export ATDM_CONFIG_USE_HWLOC=OFF
 export ATDM_CONFIG_HDF5_LIBS="-L${HDF5_ROOT}/lib;${HDF5_ROOT}/lib/libhdf5_hl.a;${HDF5_ROOT}/lib/libhdf5.a;-lz;-ldl"
-export ATDM_CONFIG_NETCDF_LIBS="-L${NETCDF_ROOT}/lib;-L${PNETCDF_ROOT}/lib;${NETCDF_ROOT}/lib/libnetcdf.a;${PNETCDF_ROOT}/lib/libpnetcdf.a;${ATDM_CONFIG_HDF5_LIBS};-lcurl"
+export ATDM_CONFIG_NETCDF_LIBS="-L${NETCDF_ROOT}/lib64;-L${PNETCDF_ROOT}/lib;${NETCDF_ROOT}/lib64/libnetcdf.a;${PNETCDF_ROOT}/lib/libpnetcdf.a;${ATDM_CONFIG_HDF5_LIBS};-lcurl"
 export ATDM_CONFIG_SUPERLUDIST_INCLUDE_DIRS=${SUPERLUDIST_ROOT}/include
 export ATDM_CONFIG_SUPERLUDIST_LIBS=${SUPERLUDIST_ROOT}/lib64/libsuperlu_dist.a
 export ATDM_CONFIG_BINUTILS_LIBS="/usr/lib64/libbfd.so;/usr/lib64/libiberty.a"

